tests/evm/test_jumpi.py

```diff
@@ -0,0 +1,133 @@
+from zkevm_specs.evm import (
+    main, Opcode, ExecutionResult, CoreState, CallState, Step,
+    Tables, RWTableTag,FixedTableTag
+)
+from zkevm_specs.util import (linear_combine, keccak256, fp_inv)
+from utils import assign_bytecode_table
+
+
+# TODO: Parametrize r, table
+def test_jumpi_nonzero_cond():
+    r = 1
+    bytecode_source = '608060406007575b00'
+    bytecode = bytes.fromhex(bytecode_source)
+    bytecode_hash = linear_combine(keccak256(bytecode), r)
+    bytecode_table = assign_bytecode_table(bytecode_source, r)
+    tables = Tables(
+        tx_table=set(),
+        bytecode_table=set(
+            bytecode_table,
+        ),
+        rw_table=set([
+            (10,  False, RWTableTag.Stack, 1, 1021, 0x07, 0, 0),
+            (11,  False, RWTableTag.Stack, 1, 1022, 0x40, 0, 0),
+        ]),
+    )
+
+    curr = Step(
+        core=CoreState(
+            rw_counter=10,
+            execution_result=ExecutionResult.JUMPI,
+            call_id=1,
+        ),
+        call=CallState(
+            is_root=False,
+            is_create=False,
+            opcode_source=bytecode_hash,
+            program_counter=6,
+            stack_pointer=1021,
+            gas_left=10,
+        ),
+        allocations=[
+            bytecode_hash, 6, 1, Opcode.JUMPI,  # bytecode lookup
+            10,  False, RWTableTag.Stack, 1, 1021, 0x07, 0, 0,  # stack pop (`dest`)
+            0x07, *31*[0],  # decompression (dest)
+            11,  False, RWTableTag.Stack, 1, 1022, 0x40, 0, 0,  # stack pop (`cond`)
+            0x40, *31*[0],
+            fp_inv(0x40),  # is_zero(cond)
+            bytecode_hash, 7, 1, Opcode.JUMPDEST,
+            0, 0, 0, 0, 0, 0, 0, 0,  # next gas_left decompression
+        ],
+        tables=tables,
+    )
+    next = Step(
+        core=CoreState(
+            rw_counter=12,
+            execution_result=ExecutionResult.JUMPDEST,
+            call_id=1,
+        ),
+        call=CallState(
+            is_root=False,
+            is_create=False,
+            opcode_source=bytecode_hash,
+            program_counter=0x07,  # `dest` value
+            stack_pointer=1023,
+            gas_left=0,
+        ),
+        allocations=[],
+        tables=tables,
+    )
+
+    main(curr, next, r, is_first_step=False, is_final_step=False)
+
+def test_jumpi_zero_cond():
+    r = 1
+    bytecode_source = '608060006007575b00'
```

Comment:
```suggestion
    # Doesn't jump to PC=7 because the condition (00) is zero. Continues to STOP.
    # PUSH1 80 PUSH1 00 PUSH1 07 JUMPI JUMPDEST STOP
    bytecode_source = '608060006007575b00'
```